src/features/settings/appearance/appearanceSlice.tsx

```diff
@@ -1,25 +1,12 @@
-import { PayloadAction, createSelector, createSlice } from "@reduxjs/toolkit";
-import { get, set } from "../storage";
-import { merge } from "lodash";
+import {
+  PayloadAction,
+  createAsyncThunk,
+  createSelector,
+  createSlice,
+} from "@reduxjs/toolkit";
 import { RootState } from "../../../store";
 import { MAX_DEFAULT_COMMENT_DEPTH } from "../../../helpers/lemmy";
-
-const STORAGE_KEYS = {
-  FONT: {
-    FONT_SIZE_MULTIPLIER: "appearance--font-size-multiplier",
-    USE_SYSTEM: "appearance--font-use-system",
-  },
-  COMMENTS: {
-    COLLAPSE_COMMENT_THREADS: "appearance--collapse-comment-threads",
-  },
-  POSTS: {
-    TYPE: "appearance--post-type",
-  },
-  DARK: {
-    USE_SYSTEM: "appearance--dark-use-system",
-    USER_MODE: "appearance--dark-user-mode",
-  },
-} as const;
```

Comment:
Correct me if I'm wrong, but wouldn't it be better to have one place where the keys are defined?

Reply:
Typescript is doing all the hard work to validate keys, so I think it's OK @rsammelson :)